fixtures/ios_app_with_sdk/Tests/AppTests.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
```

Comment:
What about describing this new fixture on the [README](https://github.com/tuist/tuist/blob/master/fixtures/README.md) file.

Reply:
👍 yikes forgot about this